src/angle/angle.c

```diff
@@ -0,0 +1,41 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   angle.c                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ycarro <ycarro@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/02/22 14:11:45 by ycarro            #+#    #+#             */
+/*   Updated: 2023/02/22 18:03:41 by ycarro           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "angle.h"
+
+void	starting_angle(t_angle *angle, t_player *player)
+{
+	if (player->dir == NORTH)
+		angle->radians = (3.0 / 2.0) * M_PI;
```

Comment:
it is better to have macros in the code and remove numbers
can you name these value? 
3.0 / 2.0
1.0 / 2.0